string deviceName
string buildDate
string serialNumber
uint32 deviceRevision

uint32    numberOfPcbs
uint32[] pcbSerialNumbers
string[] pcbNames

string imagerName
uint32 imagerType
uint32 imagerWidth
uint32 imagerHeight

string lensName
uint32 lensType
float32 nominalBaseline
float32 nominalFocalLength
float32 nominalRelativeAperture

uint32 lightingType
uint32 numberOfLights

string laserName
uint32 laserType

string motorName
uint32 motorType
float32 motorGearReduction

string apiBuildDate
uint16 apiVersion
string firmwareBuildDate
uint16 firmwareVersion

uint64 bitstreamVersion
uint64 bitstreamMagic
uint64 fpgaDna
